extensions/adobe/experience/adcloud-experienceevent.example.3.json

```diff
@@ -29,21 +28,9 @@
       {
         "id": "XA9N8wAAAMnAaj_e"
       }
-    ],
-    "CLICKID": [
-      {
-        "id": "XJm2lgAAAEmW1LVH"
-      }
     ]
   },
-  "xdm:placeContext": {
-    "xdm:geo": {
```

Comment:
Removed PlaceContext as Geo doesn't have any ID field and @id field fails when any data is ingested with @id field